pages/index.tsx

```diff
@@ -1,9 +1,10 @@
-import { Layout, siteTitle } from "../components/layout"
-import utilStyles from "../styles/utils.module.css"
-import Head from "next/head"
-import { getSortedRecipesData, RecipeData } from "../lib/recipes"
+import { Layout, siteTitle } from '../components/layout'
+import utilStyles from '../styles/utils.module.css'
+import Head from 'next/head'
+import { getSortedRecipesData, RecipeData } from '../lib/recipes'
+import { GetStaticProps } from 'next'
 
-export async function getStaticProps() {
+export const getStaticProps: GetStaticProps<HomeProps> = async () => {
```

Comment:
Not sure if I prefer the function with the regular syntax instead of the arrow function here thinking :thinking:

Reply:
In general, and in React in particular, I prefer the const syntax. It looks modernish